tests/device_tests/test_session_id_and_passphrase.py

```diff
@@ -95,24 +166,24 @@ def test_session_enable_passphrase(client):
     # We clear the session id now, so the passphrase should be asked.
     new_session_id = _init_session(client)
     assert session_id != new_session_id
-    assert _get_xpub(client, passphrase="A") == XPUB_PASSPHRASE_A
+    assert _get_xpub(client, passphrase="A") == XPUB_PASSPHRASES["A"]
 
 
 @pytest.mark.skip_ui
 @pytest.mark.setup_client(passphrase=True)
 def test_clear_session_passphrase(client):
```

Comment:
judging by the name ISTM this testcase should have been removed before? now it is duplicating `test_session_with_passphrase`